front/src/applications/operationalStudies/components/SimulationResults/SpaceTimeChart/drawTrain.ts

```diff
@@ -149,21 +164,21 @@ export default function drawTrain(
     );
   }
 
-  if (trainToDraw.allowances_headPosition && currentAllowanceSettings?.allowances) {
-    trainToDraw.allowances_headPosition.forEach((tailPositionSection) =>
-      drawCurve(
-        chart,
-        `${isSelected && 'selected'} head allowances`,
-        tailPositionSection,
-        groupID,
-        'curveLinear',
-        keyValues,
-        'allowances_headPosition',
-        rotate,
-        isSelected
-      )
-    );
-  }
+  // if (trainToDraw.allowances_headPosition && currentAllowanceSettings?.allowances) {
```

Comment:
Why did you comment this ? We should either remove it (if it is dead code) or remove the comment. Do you know what this code is used for ?
```suggestion
  // if (trainToDraw.allowances_headPosition && currentAllowanceSettings?.allowances) {
```

Reply:
I comment this because the property  `allowances_headPosition` does not exist on type of `trainToDraw` (SimulationTrain<Date>) and I haven't found anywhere where this property is used. But no, I don't know for what this is used for